automatic/dropbox/tools/Helpers.psm1

```diff
@@ -0,0 +1,25 @@
+
+function Install {
+param(
+    [string] $packageName,
+    [string] $url,
+    [string] $checksum
+)
+    $installerType = 'exe'
+    $validExitCodes = @(0)
+    $silentArgs = '/S'
+    $checksumType = 'sha256';
+
+    Install-ChocolateyPackage $packageName `
+        $installerType `
+        $silentArgs `
+        $url `
+        $url `
+        -validExitCodes $validExitCodes `
+        -Checksum64 "$checksum" `
+        -ChecksumType64 $checksumType `
+        -ChecksumType "$checksumType" `
+        -Checksum "$checksum"
+}
+
+Export-ModuleMember *
```

Comment:
The other is I would start with the most modern knowledge coming from the default template `choco new test` before attempting to improve on it. You may find that you don't even need this.

~~~powershell
$packageName= 'dropbox' 
$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"

$packageArgs = @{
  packageName   = $packageName
  fileType      = 'exe'
  url           = 'https://clientupdates.dropboxstatic.com/client/Dropbox%2014.4.19%20Offline%20Installer.exe'

  softwareName  = 'Dropbox*'
  checksum      = 'A0827474448EF4EC4E65D61E00757F7812DF9E9004B6D944FDEDEDE6D88DEFE7'
  checksumType  = 'sha256'
  silentArgs    = '/S'           # NSIS
  validExitCodes= @(0)
}

Install-ChocolateyPackage @packageArgs
~~~